composer.json

```diff
@@ -6,7 +6,7 @@
   "license": "MIT",
   "require": {
     "php": ">=7.1",
-    "php-amqplib/php-amqplib": "^2.6",
+    "php-amqplib/php-amqplib": "2.9.0",
```

Comment:
Cant we use ~ here at least?

Reply:
This is tmp until 2.10, I am not sure back-ported patches would be that necessary to worry about..also 2.9.1 is the problem..which would be grabbed by ~